.eleventy.js

```diff
@@ -77,6 +78,12 @@ module.exports = function (config) {
         }
     });
 
+    config.on("beforeBuild", () => {
+        if (!siteConfig.languages[siteConfig.defaultLanguage]) {
+            throw new Error(`The default language, ${siteConfig.defaultLanguage}, configured in src/_data/config.json is not one of your site's supported languages.`);
+        }
+    });
```

Comment:
I tried triggering this error, but it gets swallowed up. Basically the error is initially reported to the terminal, but gets removed when the webpack stuff starts up. 

_**Note**: this is only an issue with `npm start`, `npm run build` reports the errors._

It seems this is caused by: 

1. 11ty still exiting with a 0 exit code, despite the error thrown. 
2. concurrently still running other concurrent processes when one failed.

To address 1, you can add `process.exitCode = 1;` just before line 83 where the error is thrown. This tells the process to set the exit code to 1. You may want to see if another [Exit Code](https://nodejs.org/api/process.html#process_exit_codes) makes more sense. Possibly 9, but whatever seems most appropriate.

To Address 2, you can set the `--kill-others-on-fail` flag for concurrently. This tells it to end all of the other process if one exits with a non-zero exit code.

Reply:
Resolved in 3b17bb6.